### ISSUE TITLE
[Bug]: TTS & SST (Speech) configuration shows up a list of all models instead of configured ones

### ISSUE THREAD

**Issue by cfshawnmaholick** [danny-avila/LibreChat#3360]
### What happened?

Config Version (librechat.yml): 1.1.4 (1.1.5)
LibeChat Version: v0.7.3

I try to configure SST & TTS over Azure AI. If I add the `speech:` section in the configuration, LibeChat shows a list of all models (Azure or OpenAI) instead of the models configured in the config. 

If the section is added (see screenshot), the log shows missing configuration and TTS schema. 
Not sure how to handle this and documentation is not very clear here.

### Steps to Reproduce

1. Upgrade to 0.7.3 and use new config like provided in librechat.example.yml with adjusted parameters as mentioned in https://www.librechat.ai/docs/configuration/stt_tts#azure-whisper-wip
2. Run the new setup

### Relevant log output

```shell
LibreChat-TEST | 2024-07-17 07:51:06 error: Failed to get voices: Configuration or TTS schema is missing
LibreChat-TEST | 2024-07-17 07:51:07 error: Failed to get voices: Configuration or TTS schema is missing
LibreChat-TEST | 2024-07-17 07:51:09 error: Failed to get voices: Configuration or TTS schema is missing
LibreChat-TEST | 2024-07-17 07:51:14 error: Failed to get voices: Configuration or TTS schema is missing
```

### Screenshots

**Part of the configuration (1.1.4/1.1.5):**
![image](https://github.com/user-attachments/assets/c5b46e3d-c3fe-415c-926d-c0db237f6e63)

**Wrong:**
![image](https://github.com/user-attachments/assets/98841537-3ba6-4a1e-962a-566b3e31b639)

**Correct (after configuration):**
![image](https://github.com/user-attachments/assets/bfadca64-9868-4f81-968f-cf2344db5eef)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by cfshawnmaholick** [comment]
Hi, thank you very much for updating docs. 
I now used `azureOpenAI` instead of `azure` in the speech section, but the problem still exists.

**Comment by berry-13** [comment]
> Hi @berry-13, thank you very much for updating docs! I now used `azureOpenAI` instead of `azure` in the speech section, but the problem still exists.

Sorry about that, I didn't catch the whole message. Unfortunately, you're still out of luck because we still haven't released 0.7.4 yet. You can either wait or pull the latest image for now

**Comment by cfshawnmaholick** [comment]
Ok. Thank you very much! Then I will wait for 0.7.4. Have a great day!
